doc/src/manual/strings.md

```diff
@@ -469,20 +497,18 @@ but
 hello"""
 ```
 
-will contain a literal newline at the beginning. Trailing whitespace is left unaltered. They can
-contain `"` symbols without escaping. Triple-quoted strings are also dedented to the level of
-the least-indented line. This is useful for defining strings within code that is indented. For
-example:
+will contain a literal newline at the beginning.
+
+Stripping of the newline is performed after the dedentation for example:
 
 ```jldoctest
-julia> str = """
-           Hello,
-           world.
-         """
-"  Hello,\n  world.\n"
+julia> """
+         Hello,
+         world."""
+"Hello,\nworld."
 ```
 
-In this case the final (empty) line before the closing `"""` sets the indentation level.
+Trailing whitespace is left unaltered. They can contain `"` symbols without escaping.
```

Comment:
Maybe better to change "They" -> "Triple-quoted string literals" because it is a little unclear what is "they".

Reply:
fixed

Reply:
Thanks for detailed checking.